modules/counter/server-ts/sql.ts

```diff
@@ -1,11 +1,31 @@
 import knex from '../../../packages/server/src/sql/connector';
+import { Model } from 'objection';
+
+// Give the knex instance to objection.
+Model.knex(knex);
```

Comment:
Do we really need to call it here? This is app-global call, I guess, so it should be somewhere in @gqlapp/database-server-ts module.

Also what about migrations and seeds for Counter?